### ISSUE TITLE
Option tabs are not visible to company manager in Dashboard in iomad 2.8

### ISSUE THREAD

**Issue by sabdullah98** [iomad/iomad#241]
![tabs](https://cloud.githubusercontent.com/assets/11044765/8323491/e1a6ecf4-1a62-11e5-8259-e92f71c2f548.png)
 i have tried in google chrome , firefox but it is missing.
the options are accessible from the site admin>iomad block.
Tabs were working in iomad 2.7 but i have a fresh installation of 2.8 and it is not appearing to company manager but they are appearing only when i log in with site admin account.

i think there is a bug with the access rights

**Comment by ehuggett** [comment]
Same here, I've not used any version of IOMAD other than 2.8 so it was not immediately obvious (other than the 'dashboard' being less than useful for Manager roles).

The Capability checks on 64-78 of /blocks/iomad_company_admin/block_iomad_company_admin.php seem to be failing (when I comment them out the tab menu appears).

``` php
        if (iomad::has_capability('block/iomad_company_admin:companymanagement_view', $context)) {
            $tabs[1] = get_string('companymanagement', 'block_iomad_company_admin');
        }
        if (iomad::has_capability('block/iomad_company_admin:usermanagement_view', $context)) {
            $tabs[2] = get_string('usermanagement', 'block_iomad_company_admin');
        }
        if (iomad::has_capability('block/iomad_company_admin:coursemanagement_view', $context)) {
            $tabs[3] = get_string('coursemanagement', 'block_iomad_company_admin');
        }
        if (iomad::has_capability('block/iomad_company_admin:licensemanagement_view', $context)) {
            $tabs[4] = get_string('licensemanagement', 'block_iomad_company_admin');
        }
        if (iomad::has_capability('block/iomad_commerce:admin_view', $context)) {
            $tabs[5] = get_string('blocktitle', 'block_iomad_commerce');
        }
```

I've not done anything further and I'm out of time for today, hopefully tomorrow I will get a chance to check if the manager roles actually has these capabilities!

EDIT: The capabilities are in the "not set" state, by adding them to the iomad company manager roles everything seems to work as intended.

**Comment by ehuggett** [comment]
TODO add view capabilities to the manager and department manager (?) roles in local/iomad/db/install.php . This will prevent the tab menu in new installations being 'broken'.

for example https://github.com/iomad/iomad/blob/IOMAD_28_STABLE/local/iomad/db/install.php#L167

**Comment by turf212** [comment]
Fixed in 3.0
